### DISCUSSION TITLE
Get notifications back in app if swiped

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#64792] [Other Feature Feedback, Questions, & Ideas]
In the GitHub app I accidentally swiped to the side on a notification and it flew off and I can't get it back.

**Comment by arafajs** [comment]
No way to get back a notification.
You can see them from **Settings** > **Notifications**.

**Reply by jidanni** [reply]
I didn't find what you were talking about.  I'm using the app. Well at least I can change the more dangerous swipes to Save.

![Screenshot_20230824_153133_GitHub.jpg](https://github.com/community/community/assets/1230959/dbab1374-0818-428f-a385-e44f1d5b0f81)

**Reply by jidanni** [reply]
Adventure number two:


![Screenshot_20230824_153941_GitHub.jpg](https://github.com/community/community/assets/1230959/479ed117-c664-46ed-a11b-3b1ad8004d5d)

Here I am trying to mark an item as unread.

At this point I should press the blue dot at the top.

Instead I stupidly press the check mark button.

Looking at the fleeting message, it says I have marked an item as Done .

And of course there's no way to get it back, other than using a desktop computer probably.

You can't blame me. I saw a check mark and I was finished doing what I wanted to do ...so that's what happened.

(Actually I didn't do it in the item on the screenshot. Otherwise I wouldn't be talking to you here today.)
